features/support/warden.rb

```diff
@@ -0,0 +1,3 @@
+Warden.test_mode!
```

Comment:
why is this added?

Reply:
Our bad, we were prematurely adding him.